apps/src/lib/node/ledger/shell/finalize_block.rs

```diff
@@ -3543,4 +3488,83 @@ mod test_finalize_block {
             control_receiver.recv().await.expect("Test failed");
         assert_eq!(u64::from(cmd.min_confirmations), 42);
     }
+
+    /// Make a wrapper tx and a processed tx from the wrapped tx that can be
+    /// added to `FinalizeBlock` request.
+    fn mk_wrapper_tx(
```

Comment:
nit: can these helper functions go to the top of the module? Otherwise people have to read the code from the bottom up (as I currently am)